README.md

```diff
@@ -34,9 +34,13 @@ GOOGLE_PROJECT_ID
 GOOGLE_PRIVATE_KEY_ID
 GOOGLE_PRIVATE_KEY
 ```
+
+There is a template launchSettings.json.copy file in "Properties/ServiceDependencies/", all you need to do is remove the ".copy"
```

Comment:
You don't want to remove the copy, because you don't want to change that file. If you do then it will appear in git which will be annoying. Instead you should copy the file to launchSettings.json. Does that make sense? 🙂